jbdevstudio/com.jboss.jbds.central.discovery/plugin.xml

```diff
@@ -59,7 +59,7 @@ reasonable, reporting issues to these providers as required.</description>
             categoryId="org.jboss.tools.central.discovery.a.web"
             groupId="org.jboss.tools.central.discovery.a.web.core"
             certificationId="com.jboss.jbds.discovery.certification.supported"
-            description="AngularJS Tooling support for Ionic framework"
+            description="AngularJS Tooling provides content assist and navigation + Ionic framework support"
```

Comment:
Looks good now. Just a minor thing... I would use "Ionic" instead of "ionic" in all four connectors.